### DISCUSSION TITLE
Alteryx Weather Call - Null Rows

### DISCUSSION THREAD

**Discussion by jkov05** [weather-gov/api#461] [Q&A]
Hello,

I'm using Alteryx, an R-based data analytical tool, to pull monthly weather data from US counties.

![image](https://user-images.githubusercontent.com/90270434/132372509-70010558-85ed-4daf-a74a-5d0c56cb7b9f.png)

While most rows provide a proper JSON value output, some do not. When I re-run the remaining rows, I see they all output null rows and columns after the JSON Parse tool. Then, the corresponding data my team expects to have is showing all 0s. 

![image](https://user-images.githubusercontent.com/90270434/132372780-a65bf48c-bc03-4f14-bf4a-a433e97cc81a.png)

I'm confused why this is happening? Is there no weather data for that selection or is it a different issue? No link/connection errors display. I have a valid key and I accept via application/Id+json. 

Here is a sample URL for Autauga County, AL for July 2021: 
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/Jkovacs/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/Jkovacs/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#FA6E0A" vlink="#FFBE46">

https://www.ncdc.noaa.gov/cdo-web/api/v2/data?datasetid=GSOM&datatypeid=TMIN&units=standard&locationid=FIPS:01001&startdate=2021-06-01&enddate=2021-06-30
--

</body>

</html>

Any help is appreciated, thank you!

**Comment by StephenClouse** [comment]
To offer any help we would need to know what URL/endpoint you are accessing.

**Reply by jkov05** [reply]
Ah apologies. I've updated the post to include the URL for a row. It does not include the key or accept because that's input via the alteryx tool.

**Reply by StephenClouse** [reply]
Sorry, this is strictly for support of api.weather.gov. You would need to contact NCDC for help with this.
